GliaWidgets/Feature.swift

```diff
@@ -10,6 +10,7 @@ public struct Features: OptionSet {
 
     /// Bubble that is shown in engagement time outside of engagement view.
     public static let bubbleView = Self(rawValue: 1 << 0)
+    public static let secureConversations = Self(rawValue: 2 << 0)
```

Comment:
This enum was created for optional features that could be managed by the integrator. However, secure messaging is not an optional feature, so I'm not sure that `secureConversations` should be here. Or we can use this approach temporarily and remove it before release.

Reply:
Indeed I used it as a temporary measure, as this was already here. Once we merge the feature branch, then it will be removed. I have created a ticket for it https://glia.atlassian.net/browse/MOB-1707